### PR TITLE
fix(sdk)!: change Json.get() and Json.getAt() to throw

### PR DESCRIPTION
**Pull request by garysassano**
Currently `Json.get()` and `Json.getAt()` don't throw any error if the JSON property you are accessing doesn't exist.

Besides, the descriptions of the methods say that they can return `undefined`, which is not true: it's not specified in the return values and that's actually the scope of `Json.tryGet()` and `Json.tryGetAt()`.

- [x] Fix `Json.get()`
- [x] Fix `Json.getAt()`

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Wing Cloud Contribution License](https://github.com/winglang/wing/blob/main/CONTRIBUTION_LICENSE.md)*.

### CODE REVIEW

File: libs/wingsdk/src/std/json.ts [hunk 1]
```diff
@@ -146,12 +146,13 @@ export class Json {
   private constructor() {}
 
   /**
-   * Returns a specified element from the Json.
```
Comment (revitalbarletz) [inline libs/wingsdk/src/std/json.ts:149]: Do we want to keep the existing implementation and add TryGet()?

Comment (garysassano) [reply libs/wingsdk/src/std/json.ts:149]: `Json.tryGet()` and `Json.tryGetAt()` already exist.

File: libs/wingsdk/src/std/json.ts [hunk 2]
```diff
@@ -146,12 +146,13 @@ export class Json {
...
+   * Returns the value associated with the specified Json key
    *
-   * @macro ($self$)[$args$]
+   * @macro ((obj, args) => { if (obj[args] === undefined) throw new Error("Json property does not exist"); return obj[args] })($self$, $args$)
```
Comment (hasanaburayyan) [inline libs/wingsdk/src/std/json.ts:151]:
```suggestion
   * @macro ((obj, args) => { if (obj[args] === undefined) throw new Error(`Json property "${args}" does not exist`); return obj[args] })($self$, $args$)
```

**Comment by hasanaburayyan** [comment]
@garysassano also I believe we are going to want to indicate this as a breaking change

**Review by hasanaburayyan — APPROVED** [review]

**Pull request merged**